src/data/attack_animations.asm

```diff
@@ -0,0 +1,914 @@
+	dw AttackAnimation_5555 ; ATK_ANIM_96
+	dw AttackAnimation_555e ; ATK_ANIM_97
+	dw AttackAnimation_556d ; ATK_ANIM_98
+	dw AttackAnimation_5574 ; ATK_ANIM_99
+	dw AttackAnimation_557b ; ATK_ANIM_100
+	dw AttackAnimation_557e ; ATK_ANIM_101
+	dw AttackAnimation_5583 ; ATK_ANIM_102
+	dw AttackAnimation_5583 ; ATK_ANIM_103
+	dw AttackAnimation_5583 ; ATK_ANIM_104
+	dw AttackAnimation_558c ; ATK_ANIM_105
+	dw AttackAnimation_5597 ; ATK_ANIM_106
+	dw AttackAnimation_559c ; ATK_ANIM_107
+	dw AttackAnimation_55a1 ; ATK_ANIM_108
+	dw AttackAnimation_55a4 ; ATK_ANIM_109
+	dw AttackAnimation_55a9 ; ATK_ANIM_110
+	dw AttackAnimation_55b4 ; ATK_ANIM_111
+	dw AttackAnimation_55b4 ; ATK_ANIM_112
+	dw AttackAnimation_55bf ; ATK_ANIM_113
+	dw AttackAnimation_55c4 ; ATK_ANIM_114
+	dw AttackAnimation_55c9 ; ATK_ANIM_115
+	dw AttackAnimation_55ce ; ATK_ANIM_116
+	dw AttackAnimation_55d5 ; ATK_ANIM_117
+	dw AttackAnimation_55e0 ; ATK_ANIM_118
+	dw AttackAnimation_55e5 ; ATK_ANIM_119
+	dw AttackAnimation_55e6 ; ATK_ANIM_120
+	dw AttackAnimation_55ed ; ATK_ANIM_121
+	dw AttackAnimation_55f2 ; ATK_ANIM_122
+	dw AttackAnimation_55fb ; ATK_ANIM_123
+	dw AttackAnimation_55fe ; ATK_ANIM_124
+	dw AttackAnimation_5601 ; ATK_ANIM_125
+	dw AttackAnimation_5604 ; ATK_ANIM_126
+	dw AttackAnimation_5607 ; ATK_ANIM_127
+	dw AttackAnimation_560a ; ATK_ANIM_128
+	dw AttackAnimation_560f ; ATK_ANIM_129
+	dw AttackAnimation_5612 ; ATK_ANIM_130
+	dw AttackAnimation_561d ; ATK_ANIM_131
+	dw AttackAnimation_5628 ; ATK_ANIM_132
+	dw AttackAnimation_562d ; ATK_ANIM_133
+	dw AttackAnimation_5632 ; ATK_ANIM_134
+	dw AttackAnimation_5637 ; ATK_ANIM_135
+	dw AttackAnimation_5644 ; ATK_ANIM_136
+	dw AttackAnimation_564f ; ATK_ANIM_137
+	dw AttackAnimation_5654 ; ATK_ANIM_138
+	dw AttackAnimation_5659 ; ATK_ANIM_139
+	dw AttackAnimation_565e ; ATK_ANIM_140
+	dw AttackAnimation_5665 ; ATK_ANIM_141
+	dw AttackAnimation_5668 ; ATK_ANIM_142
+	dw AttackAnimation_5673 ; ATK_ANIM_143
+	dw AttackAnimation_5673 ; ATK_ANIM_144
+
+AttackAnimation_52c6: ; (6:52c6)
+	anim_player         DUEL_ANIM_GLOW
+	anim_opponent       DUEL_ANIM_HIT
+	anim_normal         DUEL_ANIM_SHAKE1
+	anim_opponent       DUEL_ANIM_SHOW_DAMAGE
+	anim_end
+
+AttackAnimation_52cf:
+	anim_player         DUEL_ANIM_GLOW
+	anim_opponent       DUEL_ANIM_8
+	anim_normal         DUEL_ANIM_SHAKE2
+	anim_opponent       DUEL_ANIM_SHOW_DAMAGE
+	anim_end
+
+AttackAnimation_52d8:
+	anim_player         DUEL_ANIM_GLOW
+	anim_opponent       DUEL_ANIM_THUNDER_SHOCK
+	anim_opponent       DUEL_ANIM_HIT
+	anim_normal         DUEL_ANIM_SHAKE1
+	anim_opponent       DUEL_ANIM_SHOW_DAMAGE
+	anim_end
+
+AttackAnimation_52e3:
+	anim_player         DUEL_ANIM_GLOW
+	anim_opponent       DUEL_ANIM_11
+	anim_opponent       DUEL_ANIM_12
+	anim_opponent       DUEL_ANIM_HIT
+	anim_normal         DUEL_ANIM_SHAKE1
+	anim_opponent       DUEL_ANIM_SHOW_DAMAGE
+	anim_end
+
+AttackAnimation_52f0:
+	anim_player         DUEL_ANIM_GLOW
+	anim_normal         $65

```

Comment:
Do these few hard-coded values mean something different?

Reply:
Animation indices >$60 are handled differently and are not stored in the regular map pointer tables.